tests/setup/jestSetup.js

```diff
@@ -3,7 +3,7 @@ require('dotenv').config();
 const PAGE_TITLE = 'Plug';
 const EXTENSION_PATH = require('path').join(__dirname, '..', '..', 'extension', 'chrome');
 
-jest.setTimeout(60000); // in milliseconds
+jest.setTimeout(600000); // in milliseconds
```

Comment:
About this, we can change it back to how it was originally, and for the test we can use the command `page.setDefaultTimeout(timeout)` in the test you needed, the `page.setDefaultTimeout(timeout)` is located prior the line that will take long time wait (I think) try after, just in case I am wrong